examples/avr-rss2/AES128HW_test/AES128HW_test.c

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2015, Copyright Robert Olsson 
```

Comment:
Is the double occurrence of "Copyright" intentional? Also the project-conf.h file below has it.